cmd/edit/logforwarder/cmd.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logforwarder
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	short = "Edit a log forwarder for a cluster"
+	long  = "Edit a log forwarder configuration for a cluster. A cluster ID must be provided, as well as " +
+		"a valid log forwarder ID on that cluster. For example: 'rosa edit log-forwarder -c my-cluster-1 " +
+		"2n4b8f8ai80cs6kmjmdgqlqplh73r411'"
+)
+
+var (
+	logFwdConfig string
+)
+
+func NewEditLogForwarderCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "log-forwarder -c <cluster-id> <log-fwd-id>",
```

Comment:
should this be something like below?
log-forwarder -c <cluster-id> <log-fwd-id> --log-fwd-config <log-fwd-config-path>